src/main/scripts/createAppGatewayIngress.sh

```diff
@@ -0,0 +1,255 @@
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I would like to understand why you are not doing the `set -Eeou pipefail ` best practice here?

Reply:
See same reasons from [the comment](https://github.com/WASdev/azure.liberty.aks/pull/46#discussion_r899781069) below.